app/views/users/users/_form.html.slim

```diff
@@ -8,12 +8,12 @@
   .form-row
     .col-md-6= f.input :birth_name, placeholder: 'Nom de naissance'
     .col-md-6= date_input(f, :birth_date)
-  = f.input :phone_number, as: :tel
+  = f.input :phone_number, as: :tel, required: local_assigns[:rdv_wizard]&.motif&.phone?
   label Préférences de notifications
   .d-flex
     div= f.input :notify_by_email
     div.ml-3= f.input :notify_by_sms
-  - address_value = local_assigns[:rdv_wizard].present? && user.address.nil? ? rdv_wizard[:where] : user.address
+  - address_value = local_assigns[:rdv_wizard].present? && user.address.nil? ? rdv_wizard.to_query[:where] : user.address
```

Comment:
en passant je note qu'on pourrait eviter de passer par to_query ici meme si ce n'est pas toi qui l'a introduit

Reply:
J'en profite pour corriger, merci !

Reply:
Finalement non, c'est pas possible de l'enlever. Je le remets. C'est ça qui fait exploser le test actuellement.

Reply:
oui je voulais dire qu'il faudrait rajouter un `delegate :where, to :attributes` ou qqch comme ca dans le modèle pour pouvoir faire `rdv_wizard.where` plutot que passer par la serialisation en to_query puis l'acces a la clé where

C'est un detail donc pas de soucis pour faire ca plus tard